static/js/impala/listing.js

```diff
@@ -10,7 +10,11 @@ $(function() {
             // L10n: {0} is an app name.
             var msg = format(gettext('This theme is incompatible with your version of {0}'),
                              [z.appName]);
-            $this.append(format('<span class="notavail">{0}</span>', msg));
+            if ($('body').hasClass('restyle')) {
+              $this.append(format('<button class="disabled concealed notavail">{0}</button>', msg));
```

Comment:
Actually this is just totally the wrong thing and I shouldn't have added it. In my defense, these commits were made without coffee I think. :disappointed: I'll remove it.